README.md

```diff
@@ -168,9 +101,6 @@ with nGraph backends.
 
 Please add the following line to enable nGraph: `import ngraph_bridge`
 
-Note: The actual filename for the pip package may be different as it's version 
-dependent. Please check the `build/python/dist` directory for the actual pip wheel.
-
```

Comment:
Line 176:
`nGraph is being added to the TensorFlow source tree the using pull requests from 
 time to time`

extra `the`

How about:
`nGraph is updated in the TensorFlow source tree using pull requests from time to time`